layout/footer.php

```diff
@@ -1,76 +1,81 @@
+<?php
+$database  = new Database();
+$query = "SELECT * FROM services";
+$services = $database->query($query);
+$database->close();
+?>
+
 <!-- Footer Start -->
+
 <div class="footer">
-                <div class="container">
-                    <div class="row">
-                        <div class="col-md-6 col-lg-4">
-                            <div class="footer-about">
-                                <h2>About Us</h2>
-                                <p>
-                                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque eu lectus a leo tristique dictum nec non quam. Suspendisse convallis, tortor eu placerat rhoncus, lorem quam iaculis felis, sed eleifend lacus neque id eros. Integer convallis volutpat neque
-                                </p>
-                            </div>
-                        </div>
-                        <div class="col-md-6 col-lg-8">
-                            <div class="row">
-                        <div class="col-md-6 col-lg-4">
-                            <div class="footer-link">
-                                <h2>Services Areas</h2>
-                                <a href="">Civil Law</a>
-                                <a href="">Family Law</a>
-                                <a href="">Business Law</a>
-                                <a href="">Education Law</a>
-                                <a href="">Immigration Law</a>
-                            </div>
+    <div class="container">
+        <div class="row">
+            <div class="col-md-6 col-lg-4">
+                <div class="footer-about">
+                    <h2>About Us</h2>
+                    <p>
+                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque eu lectus a leo tristique dictum nec non quam. Suspendisse convallis, tortor eu placerat rhoncus, lorem quam iaculis felis, sed eleifend lacus neque id eros. Integer convallis volutpat neque
+                    </p>
+                </div>
+            </div>
+            <div class="col-md-6 col-lg-8">
+                <div class="row">
+                    <div class="col-md-6 col-lg-4">
+                        <div class="footer-link">
+                            <h2>Services Areas</h2>
+                            <?php foreach($services as $serve): ?>
```

Comment:
```suggestion
                            <?php foreach($services as $serve): ?>
```
change `$serve` to `$service`